package/scripts/presto_client.py

```diff
@@ -80,7 +80,8 @@ def ensure_nodes_are_up(client):
         raise RuntimeError('Presto server failed to start within \
 {0} seconds.'.format(RETRY_TIMEOUT))
 
-
+# This class was copied more or less verbatim from
```

Comment:
Yes that is the next step in the integration. I need to figure out how to extract the topology information the user sets during setup and pass it down to these functions.